### DISCUSSION TITLE
How do I integrate this into docusaurus

### DISCUSSION THREAD

**Discussion by flamecopper** [hubtype/botonic#2497] [General]
Hi there, as the topic suggest. How do I integrate this into docusarus?

**Comment by flamecopper** [comment]
Should i use the standalone js or is there a way to directly import the component into it?

**Comment by manuelalferez** [comment]
To better understand your objectives, could you please provide more details about what you are trying to achieve and the specific problems you are encountering? This will help us better assist you with your inquiry. 
Thanks!
